VMEncryption/main/ExtensionParameter.py

```diff
@@ -174,6 +174,13 @@ def _is_kv_equivalent(self, a, b):
             if b[-1] == '/': b = b[:-1]
         return a==b
 
+    def cmk_changed(self):
+        if (self.KeyEncryptionKeyURL or self.get_kek_url()) and \
```

Comment:
Why does this check the KeyEncryptionKeyUrl twice?

Reply:
one is locally stored and other is from public setting.